packages/litexa/src/parser/litexa.pegjs

```diff
@@ -2198,9 +2197,32 @@ For example, if a handler in the skill looks like this:
 Then the following statements behave the same way in your test:
 
 ```coffeescript
-user: "my name is Cat" # Litexa deduces $name = Cat from the utterance
-user: NameIntent with $name = Cat # append more slots by separating them with commas
-user: "my name is" with $name = Cat # this is valid, but can't happen in a real interaction
+user: "my name is Cat" # Litexa deduces $name = "Cat" from the utterance
+user: NameIntent with $name = "Cat" # append more slots by separating them with commas
```

Comment:
oops, can you complete the example here? Maybe something like:

NameIntent with $name="Cat", $surname="Dog"

Reply:
Updated example to use 2 slots, and reformulated a bit.